frontend/src/features/properties/filter/PropertyFilterOptions.tsx

```diff
@@ -9,16 +9,12 @@ import { IPropertyFilter } from './IPropertyFilter';
 export const PropertyFilterOptions: React.FC = () => {
   const state: { options: any[]; placeholders: Record<string, string> } = {
     options: [
+      { label: 'Building Name', value: 'buildingName' },
       { label: 'Address', value: 'address' },
-      { label: 'Location', value: 'administrativeArea' },
```

Comment:
Wanted to confirm this was correct to remove since we have it in the new filter?